include/d/d_event_manager.h

```diff
@@ -68,6 +68,7 @@ class dEvent_manager_c {
     int setStartDemo(int eventInfoIdx) { return mException.setStartDemo(eventInfoIdx); }
     void cancelStartDemo() { /* TODO */ }
     void setCameraPlay(u32 v) { mCameraPlay = v; }
+    int getCameraPlay() { return mCameraPlay; }
```

Comment:
Slightly wrong name, should be `cameraPlay()`
FYI, when adding inlines you can just copy paste all of the already-demangled names out of the list of all weak functions pinned in Discord so you don't have to manually try to demangle them one by one (or use objdiff):
```
dEvent_manager_c::cameraPlay()
dEvent_manager_c::cancelStartDemo()
dEvent_manager_c::dEvent_manager_c()
dEvent_manager_c::getEventHIO()
dEvent_manager_c::getFlags()
dEvent_manager_c::setCameraPlay(int)
dEvent_manager_c::setStartDemo(int)
dEvent_manager_c::~dEvent_manager_c()
```
Also, whoever added `setCameraPlay` gave it the wrong param type (u32 vs int)